tests/thread_stats.c

```diff
@@ -0,0 +1,80 @@
+#include <thread.h>
+#include <ktest.h>
+#include <klog.h>
+#include <sched.h>
+#include <runq.h>
+
+#define THREADS_NUMBER 10
+#define TEST_TIME 500
+
+static void thread_nop_function(void *arg) {
+  while (tv2st(get_uptime()) < TEST_TIME + tv2st(*(timeval_t *)arg))
```

Comment:
Why so complicated? Isn't it just better to calculate everything using `timeval_t` type?